src/main/java/codeu/model/store/basic/MessageStore.java

```diff
@@ -103,12 +103,3 @@ public void setMessages(List<Message> messages) {
 	 return sentMessages;
   }
 }  
-  
-  
-=======
-  public void deleteLastMessage(List<Message> messages) {
```

Comment:
this was my function but changed it to a more general just deleteMessage function so i don't think this is needed. Maybe just pull from master one more time and it will show up